x-pack/legacy/plugins/actions/index.ts

```diff
@@ -27,6 +27,14 @@ export function actions(kibana: any) {
       return Joi.object()
         .keys({
           enabled: Joi.boolean().default(false),
+          whitelistedHosts: Joi.alternatives()
+            .try(
+              Joi.array()
+                .items(Joi.string().hostname())
+                .sparse(false),
+              Joi.string().valid('any', 'none')
```

Comment:
Do we need `none`?  How is that different from an empty array?  I'm not sure if there's an existing convention for specifying the "any" concept in es/kibana config - maybe `*`?

Reply:
The way I see it, an empty array essentially works as a side effect of the implementation, while 'none' makes it much more explicit.
That said, I found the _requestHeadersWhitelist_ field explicitly states you should use an empty array for none.... so we can align with that standard.

Regarding '*' vs 'any', if there's a standard then obviously I'll align with it, but I can't find an example of an equivalent config value. Any ideas?

Reply:
OK, I take that back - found an example of a config that uses '*' to denote _all_, so I'll go with it.